docs/docs-content/clusters/data-center/nutanix/nutanix.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: "Nutanix"
+title: "Nutanix"
+description: "Learn how to configure Nutanix and create Nutanix clusters in Palette."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["data center", "nutanix"]
+---
+
+
+[Nutanix](https://www.nutanix.com/what-we-do) is a hyper-converged infrastructure platform that combines storage, compute, and networking into a single integrated system. Palette provides a generic framework built upon the open-source [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io) initiative to support adding cloud providers. As a Tech Preview of implementing this framework, you can deploy Kubernetes clusters on Nutanix using Palette. Integration with Nutanix is achieved through a Self-hosted Private Cloud Gateway (PCG), which establishes a secure connection with Nutanix Prism Central and allows Palette to monitor Nutanix clusters.
+
+## Get Started
+
+Learn how to deploy a cluster to Nutanix by using Palette. Check out the [Deploy a Cluster with Palette](../../public-cloud/deploy-k8s-cluster.md) tutorial to get started.
+
+The section below describes the workflow to register Nutanix with Palette and deploy a cluster. 
+
+## Workflow
+
+The following summary steps outline the Nutanix cloud workflow illustrated in the diagram. Since only self-hosted instances of Palette and Palette VerteX have access to the system console by a [system administrator](../../../glossary-all.md#system-administrator), the registration step is performed at the system level, making the Nutanix cloud available to all the tenants. Regular Palette or VerteX users perform the remaining steps in Palette and their local environment.
+
+![Diagram showing the overall Nutanix workflow for users to construct self-hosted cloud solutions tailored to their environment.](/clusters_data-center_nutanix_workflow.png)
+
+1. A [system administrator](../../../glossary-all.md#system-administrator) registers Nutanix with Palette by preparing YAML templates to configure the cluster and node pools and invokes APIs to define the logo, required cloud account information and key-value pairs to create the Nutanix cloud account. Palette renders a default logo if no logo is specified. 
+
+  :::caution
+  To make the Palette-provided **Nutanix CSI ---link to Nutanix CSI pack page that is being created---** pack available to users during profile creation, the Nutanix cloud must be registered using the name `nutanix`. For guidance in using registration APIs and providing required variable values, review [Register the Cloud](register-nutanix-cloud.md#register-the-cloud).
```

Comment:
@caroldelwing I think we can remove this info box from this high level page as you have a similar one in the `Register Nutanix Cloud` page